### ISSUE TITLE
Write blog post with a follow up for "Auto-cancelling builds ..."

### ISSUE THREAD

**Issue by humitos** [readthedocs/blog#185]
Blog post: https://blog.readthedocs.com/cancel-old-builds/

It would be good to update this blog post, or maybe better, write a new one with some updated numbers. Now, we know that **around 10% of total builds per day are cancelled**, which is a really good number to communicate. We could also related this to

- savings in $$ if we find a good metric to get this number
- CPU time saved (comparison between total build time per day) [^1]

| Builds cancelled per day  | Build time per day |
| ------------- | ------------- |
| ![Screenshot_2022-10-21_19-47-55](https://user-images.githubusercontent.com/244656/197258034-6ee9fb6a-07a1-47e8-8fde-e78bff7d3e13.png)  | ![Screenshot_2022-10-21_19-48-14](https://user-images.githubusercontent.com/244656/197258053-f845f9f6-78db-4d6e-a7d2-29b90339300e.png)  |

[^1]: However, this involves platform growth, so it may not be representative. We could do "avg build time for version * amount of builds cancelled" to get a more accurate number probably.

**Comment by benjaoming** [comment]
Perhaps this post can also contain something about https://github.com/readthedocs/readthedocs.org/pull/9649 ?

**Comment by humitos** [comment]
Yes, or writing another blog post for that particular feature + stats.

We may need to update the Metabase queries and probably add a new one to track cancelled builds because of the new exit code.

**Comment by humitos** [comment]
I created this Metabase question that will track conditionally cancelled builds: https://ethicalads.metabaseapp.com/question/315-conditionally-cancelled-builds-exit-183-per-day
